### ISSUE TITLE
Improve legibility of coloured notes

### ISSUE THREAD

**Issue by HelenWDTK** [mysociety/alaveteli#8277]
Currently the notes use a text colour that matches the colour of the note, but sometimes this is difficult to read e.g green text on a green background. We should change the text colour to black, or something else that ensures that there is sufficient contrast to make this easy to read.

**Comment by garethrees** [comment]
Moved to mysociety/alaveteli since these styles are built in.

Assessment of current styles:

* Blue: https://webaim.org/resources/contrastchecker/?fcolor=006687&bcolor=EDFBFF (6.1:1 – pretty good)
* Yellow: https://webaim.org/resources/contrastchecker/?fcolor=D3710E&bcolor=FDF0E3 (3.05:1 – bad)
* Green: https://webaim.org/resources/contrastchecker/?fcolor=358753&bcolor=CFECDA (3.51:1 – bad)
* Red: https://webaim.org/resources/contrastchecker/?fcolor=D32A0E&bcolor=FDE7E3 (4.31:1 – bad)
